lib/err.c

```diff
@@ -63,7 +63,17 @@ tsi_strerror(int err)
             ret = "Bad path information: right > num_sites";
             break;
         case TSI_ERR_MATCH_IMPOSSIBLE:
-            ret = "Match impossible under input parameters.";
+            ret = "Unexpected failure to find matching haplotype; please open "
+                  "an issue on GitHub";
+            break;
+        case TSI_ERR_MATCH_IMPOSSIBLE_EXTREME_MUTATION_PROBA:
+            ret = "Cannot find match: the specified mutation probability is "
```

Comment:
Since this is in tsinfer, not tskit, should we be calling these mismatch probabilities, rather than mutation probabilities?